docs/release-notes/june-2024.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: June-2024
+hide_title: true
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: June 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for June 2024
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+- The Semgrep **Jira integration** is now in **public beta**. Create Jira tickets from Semgrep AppSec Platform to your Jira projects and configure mappings from Semgrep fields to Jira fields. Read the [Jira integration documentation](/semgrep-appsec-platform/jira#enable-the-jira-integration) to learn more.
+  - Assistant remediation guidance is now available in Jira tickets you create. <!-- 14994 -->
+  - Added a red Jira ticket icon in the findings page to make it clear when Jira ticket creation fails.  <!-- 14835 -->
```

Comment:
We don't currently, but I wouldn't be opposed to standardizing how we write these lists (e.g., use the "added a thing" format for most things unless it's a major feature or whatever). Mostly, we use the past tense because it's work that's done, though how we structure the note can vary. WDYT? 

cc @s-santillan

Reply:
Yeah I think past tense can work for most things. When it’s something like “Jira integration is now in beta” that might feel forced if put into a past tense like “Jira integration was made available in public beta” unless there’s a more elegant way to write that.

Reply:
Happy to follow:
- Past tense for most things (Added a button...)
- Present tense for launch items (so and so is now in beta)